.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "modules/try-convert"]
 	path = modules/try-convert
-	url = https://github.com/dotnet/try-convert
```

Comment:
This needs to be fixed when the try-convert PR merges in
https://github.com/dotnet/try-convert/pull/434